### PR TITLE
add Review Button

### LINKED ISSUE #32: As a lecturer, I can access a lecture after it ended. (Estimation 3)
**FROM THE LECTURERS VIEW**

- Ending a lecture adds the option "review" next to it in the [lectures list of lectures](https://www.figma.com/file/78SUuNyEDnrKGGeJPmvnmp/Paper-Prototype-%231?node-id=0%3A1)

- Clicking on "review" takes the lecturer to the lectures dashboard

_Hint:
Use the [current paper prototype](https://www.figma.com/file/78SUuNyEDnrKGGeJPmvnmp/Paper-Prototype-1?node-id=0%3A1) as guidance, but remember that there will be a new version with a dashboard by wednesday._

### PR DESCRIPTION
**Pull request by adrianjost**
# Resolves #32

This PR adds an Review Button for ended lectures

Acceptance Criteria:
- [x] Ending a lecture adds the option "review" next to it in the [lectures list of lectures](https://www.figma.com/file/78SUuNyEDnrKGGeJPmvnmp/Paper-Prototype-%231?node-id=0%3A1)
- [x] Clicking on "review" takes the lecturer to the lectures dashboard

Testing steps:
- end a lecture

image

---

Definition Of Done:
- [x] ~[Class-Diagram](https://www.lucidchart.com/invitations/accept/705a122b-58a9-4dfa-8b16-c936bc0a46bc) updated~ nothing to update

### CODE REVIEW

File: spec/views/lectures/index.html.erb_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "lectures/index", type: :view do
   before(:each) do
-    assign(:lectures, [
+    @lectures = assign(:lectures, [
```
Comment (essal0815) [inline spec/views/lectures/index.html.erb_spec.rb]: Could/Should we use here a factory bot ?
- [ ] Check if usage of factorybot would be better, if yes implement

File: app/assets/stylesheets/lectures.scss [hunk 2]
```diff
@@ -1,3 +1,7 @@
 // Place all the styles related to the lectures controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+
```
Comment (essal0815) [inline app/assets/stylesheets/lectures.scss]: I just believe in your design and UI skills, I have no clue of css ;)

File: spec/views/lectures/index.html.erb_spec.rb [hunk 3]
```diff
@@ -24,4 +24,26 @@
     assert_select "tr>td", text: "Enrollment Key".to_s, count: 2
     assert_select "tr>td", text: "created".to_s, count: 2
   end
+
+  it "renders Start button for created lectures" do
+    render
+    assert_select "[value='Start']", count: 2
+  end
+
+  it "renders View and End button for running lectures" do
```
Comment (essal0815) [inline spec/views/lectures/index.html.erb_spec.rb]: Here you set the status directly. Shouldn't be also checked that, the button click changes the status? Or is this already checked by an other test I just didn't see?
- [ ] Check if this behavior is already tested

Comment (adrianjost) [reply spec/views/lectures/index.html.erb_spec.rb]: View tests should not test behaviour, only the html output. It's also impossible to click buttons in html spec tests.
The Tests for Ending a Lecture where implemented in #38 where this feature is coming from.

File: spec/views/lectures/index.html.erb_spec.rb (hunk 1 above)
Comment (adrianjost) [reply spec/views/lectures/index.html.erb_spec.rb]: thanks, I am using the factory bot now

**Pull request merged**